tests-e2e/cypress/integration/channels/rhs/status_update_spec.js

```diff
@@ -242,4 +242,30 @@ describe('channels > rhs > status update', () => {
             });
         });
     });
+
+    describe('playbook with disabled status updates', () => {
+        before(() => {
+            // # Create a public playbook
+            cy.apiCreatePlaybook({
+                teamId: testTeam.id,
+                title: 'Playbook',
+                userId: testUser,
+                broadcastChannelId: testChannel.id,
+                statusUpdateEnabled: false,
+            }).then((playbook) => {
+                testPlaybook = playbook;
+            });
+        });
+
+        describe('Omit status update dialog when status updates ar disabled', () => {
```

Comment:
Typo (and I think we always start with lowercase):
```suggestion
        describe('omit status update dialog when status updates are disabled', () => {
```